biosdk-client/pom.xml

```diff
@@ -407,6 +408,51 @@
 			</build>
 		</profile>
 	</profiles>
+
+	<repositories>
+		<repository>
+			<id>spring-releases</id>
+			<url>https://repo.spring.io/release</url>
+		</repository>
+		<repository>
+			<id>spring-milestones</id>
+			<url>https://repo.spring.io/milestone</url>
+		</repository>
+		<repository>
+			<id>spring-snapshots</id>
+			<url>https://repo.spring.io/snapshot</url>
+		</repository>
```

Comment:
do we need all the above repositories ? Central maven will not have all the dependencies that is required ?

Reply:
yes certain times it happens

Reply:
Normally we get all the dependencies between certal maven and sona, if remaining is not absolutely requried, can we just remove the same ?

Reply:
removed and tested